src/applications/vaos/tests/mocks/setup.js

```diff
@@ -156,32 +166,65 @@ export async function setVAFacility(store, facilityId) {
       parentStationCode: siteCode,
     },
   };
-  mockParentSites([siteCode], [parentSite]);
-  const facilities = [
+  const directFacilityAttributes = getDirectBookingEligibilityCriteriaMock()
+    .attributes;
+
+  const directFacilities = [
```

Comment:
Looking at this again, I think we can simplify how we create both `directFacilities` and `requestFacilities`.

```
const directFacilities = [
  getDirectBookingEligibilityCriteriaMock({ id: facilityId, typeOfCareId }),
];
const requestFacilities = [
  getRequestEligibilityCriteriaMock({ id: facilityId, typeOfCareId }),
];
 ```

Reply:
replaced  simplified object constants